DZ_7/Task_34.py

```diff
@@ -0,0 +1,51 @@
+# Задача 34:  Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм.
+# Поскольку разобраться в его кричалках не настолько просто, насколько легко он их
+# придумывает, Вам стоит написать программу.
+# Винни-Пух считает, что ритм есть, если число слогов
+# (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. Фраза может состоять
+# из одного слова, если во фразе несколько слов, то они разделяются дефисами.
+# Фразы отделяются друг от друга пробелами. Стихотворение  Винни-Пух вбивает в программу с клавиатуры.
+# В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все
+# не в порядке
+#
+# *Пример:*
+#
+# **Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да
+#     **Вывод:** Парам пам-пам
+import sys
+
+verse = input('Напиши стишок: ').split()
+
+
+# def split(i):
+#     return [char for char in i]
+
+# Проверка на отстутвие пробелов
+if len(verse) == 1:
+    print('Это не стишок')
+    sys.exit()
+
+
+# Решение задачи
+
+# считаем гласных
+
+new_list = []
+
+for i in verse:
+    count = 0
+    for j in list(i):
+        if j == 'а' or j == 'я' or j == 'у' or j == 'ю' or j == 'о' or j == 'е' or j == 'ё' or j == 'э' or j == 'и' or j == 'ы':
```

Comment:
Александр я уверен, что выполняя это задание у вас возникла мысль, что возможно, существует более коротких способ проверить это условие. И он действительно существует
```suggestion
        if j in 'аяуюоеёэиы':
```